### PR TITLE
add clean up membership in autopush hub

### PR DESCRIPTION
**Pull request by liwenhao0810**
Add autopush-gkehub endpoint for cleaning up memberships.

### CODE REVIEW

**Comment by chizhg** [comment]
@alvaroaleman @stevekuznetsov do you mind taking a look at this PR? Thanks in advance!

**Comment by alvaroaleman** [comment]
I am not familiar with the GCP janitor, sorry
/assign @ixdy

File: cmd/janitor/gcp_janitor.py [hunk 1]
```diff
@@ -97,6 +97,11 @@
     # staging GKE hub memberships
     'staging-gkehub.sandbox.googleapis.com': [
         Resource('', 'container', 'hub', 'memberships', None, None, False, False, None),
+    ],
+
+    # staging GKE hub memberships
```
Comment (ixdy) [inline cmd/janitor/gcp_janitor.py:102]: nit: staging -> autopush?

File: cmd/janitor/gcp_janitor.py [hunk 2]
```diff
@@ -558,7 +563,7 @@ def main(project, days, hours, filt, rate_limit, service_account, additional_zon
     for api, resources in RESOURCES_BY_API.items():
         if not api_enabled(project, api):
             continue
-        if api == 'staging-gkehub.sandbox.googleapis.com' or api == 'gkehub.googleapis.com':
+        if api == 'staging-gkehub.sandbox.googleapis.com' or api == 'gkehub.googleapis.com' or api == 'autopush-gkehub.sandbox.googleapis.com':
```
Comment (ixdy) [inline cmd/janitor/gcp_janitor.py:566]: this is basically fine, can it be simplified somehow?

Comment (liwenhao0810) [reply cmd/janitor/gcp_janitor.py]: Added a set for gkehub apis, not an expert in python, please check :).

**Review by ixdy — APPROVED** [review]

**Pull request merged**